src/components/find-on-shelf/FindOnShelfModal.tsx

```diff
@@ -26,12 +27,16 @@ export interface FindOnShelfModalProps {
   manifestations: Manifestation[];
   workTitles: string[];
   authors: Work["creators"];
+  selectedPeriodical: GroupListItem | null;
+  setSelectedPeriodical: (selectedPeriodical: GroupListItem) => void;
```

Comment:
:+1: for using the existing `GroupListItem` type.

Consider renaming it to something more related to periodicals or perhaps or take inspiration from the [`Periodical` type](https://github.com/reload/dpl-react/blob/b13a4fb4fa6406b4d47ce9fabf02a2171898fa64/src/components/reservation/helper.ts#L55-L55) from the material component.

Reply:
Thanks! :) 
And I agree - GroupListItem is quite vague. I renamed it to PeriodicalEdition.